superbuild/CMakeLists.txt

```diff
@@ -32,29 +32,47 @@ check_cache_var_declaration(L0_TAG "Level Zero url")
 check_cache_var_declaration(ISPC_CORPUS_URL "ispc-corpus url")
 
 set(GNUWIN32 "c:/gnuwin32" CACHE STRING "Set path to gnuwin32 root dir (Windows only)")
-set(CCACHE_VERSION "4.7.4" CACHE STRING "ccache version ")
+set(CCACHE_VERSION "4.8.2" CACHE STRING "ccache version ")
 set(CCACHE_DIR "default" CACHE STRING "ccache cache directory")
+set(PREBUILT_STAGE1 "NO" CACHE STRING "Provide pre-built stage1 archive to skip stage1 jobs")
+set(PREBUILT_STAGE1_PATH "NO" CACHE STRING "Provide pre-built stage1 toolchain path")
 set(PREBUILT_STAGE2 "NO" CACHE STRING "Provide pre-built stage2 archive to skip pre ispc-stage2 jobs")
 set(PREBUILT_STAGE2_PATH "NO" CACHE STRING "Provide pre-built stage2 toolchain path")
+set(STAGE1_TOOLCHAIN_INSTALL_PREFIX "NO" CACHE STRING "Path to install stage1 Clang/LLVM to")
 set(STAGE2_TOOLCHAIN_INSTALL_PREFIX "NO" CACHE STRING "Path to install stage2 Clang/LLVM to")
+set(STAGE2_ISPC_INSTALL_PREFIX "NO" CACHE STRING "Path to install stage2 ISPC to")
```

Comment:
Should the default values here be `"NO"` if they are paths?

Reply:
They are strings, and "NO" here means that no paths have been provided.